analyzer/src/main/kotlin/managers/CocoaPods.kt

```diff
@@ -46,7 +86,206 @@ class CocoaPods(
         ) = CocoaPods(managerName, analysisRoot, analyzerConfig, repoConfig)
     }
 
+    private val podspecCache: MutableMap<String, Podspec> = mutableMapOf()
+
+    override fun command(workingDir: File?) = "pod"
+
+    override fun getVersionRequirement(): Requirement = Requirement.buildIvy("[1.10.1,)")
+
+    override fun getVersionArguments(): String = "--version --allow-root"
```

Comment:
Nit: I believe it'd be fine to also omit the explicit return types here, like in the functions above and below, as it's very clear from the function expression.

Reply:
This comment also referred to line 93.